tests/__main__.py

```diff
@@ -1,3 +1,7 @@
+import sys
+if sys.version_info < (3, 5):
+    print ("Only support python version >= 3.5")
+    exit(-1)
```

Comment:
This should be `exit(1)`, ref: [exit code](http://tldp.org/LDP/abs/html/exitcodes.html)

And also need a newline after that.

Reply:
Done